MC-38191: Implement check for patch availability in MQP

bubasuma opened the pull request:
**Refactor code to use composer autoload with psr4**

- Add namespace to classes
- Use composer autoload to load files

**MC-38191: Implement check for patch availability in MQP**

bubasuma committed:
Refactor code to use composer autoload with psr4

- Add namespace to classes
- Use composer autoload to load files

bubasuma committed:
MC-38191: Implement check for patch availability in MQP

viktym commented on app/config/magento_package_versions.json:
```diff
@@ -0,0 +1,135 @@
+  "2.4.1": {
+    "magento/magento2-base": "2.4.1",
+    "magento/magento2-ee-base": "2.4.1",
+    "magento/magento2-b2b-base": "1.2.1",
```
```suggestion
    "magento/magento2-b2b-base": "1.3.0",
```

bubasuma committed:
Improve user experience

- Display MQP version
- Change No to Fail
- Fix styling
- Fix b2b package version

bubasuma committed:
Change MQP result format

bubasuma committed:
Fix MQP 'patch not found' error message

viktym commented on design/js/script.js:
```diff
@@ -1,14 +1,123 @@
+                        output += '<td class="' + falseResultClass + '">No';
+                    }
+                    if (release.result['git_apply'] === 1) {
+                        output += '<td class="td_ok">Ok';
```
Please replace 'Ok' => 'Yes'

bubasuma replied:
Done

bubasuma committed:
Change 'ok' to 'yes'

viktym commented:
Looks good to me

bubasuma committed:
Remove fixed width from table columns
